test/e2e/framework/deployment_util.go

```diff
@@ -36,11 +36,6 @@ import (
 	imageutils "k8s.io/kubernetes/test/utils/image"
 )
 
-// UpdateDeploymentWithRetries updates the specified deployment with retries.
```

Comment:
So I would lift this whole file under framework/deployment/foo.go 

I wouldn't rip out just the single function.

Reply:
Awesome, this makes sense. I was wondering whether to do what I was doing here (trying to reduce the library calls) or to better organize it within a pkg (what you mentioned).